jetty-integrations/jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionDataStore.java

```diff
@@ -159,6 +157,8 @@ public class MongoSessionDataStore extends NoSqlSessionDataStore
      */
     private DBObject _version1;
 
+    private final Pattern _workerNamePattern = Pattern.compile("[0-9a-zA-Z]*");
```

Comment:
Not that it will make any difference, but:
```suggestion
    private final static Pattern __workerNamePattern = Pattern.compile("[0-9a-zA-Z]*");
```

Reply:
Done.